### ISSUE TITLE
Progress Bar Examples have accessibility issues

### ISSUE THREAD

**Issue by coliff** [twbs/bootstrap#36714]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

The provided examples at: https://getbootstrap.com/docs/5.2/components/progress/ have accessibility issues reported by both Google Lighthouse and Webhint.io

I think that the progress bar should have a non-empty `aria-label` attribute or a `title`.

REF:
https://dequeuniversity.com/rules/axe/4.4/aria-progressbar-name?application=axeAPI

Let me know what you think, I'd be happy to open a PR...

(I realise this might be flagged as an accessibility issue incorrectly and maybe the `aria-valuenow`, `aria-valuemax` and `aria-valuetext` attributes are enough? cc @patrickhlauke )

### Reduced test cases

Webhint screenshot:
![image](https://user-images.githubusercontent.com/1212885/178211544-262ab1b9-9c2c-4750-9620-ab7e4c52613e.png)

Lighthouse screenshot:
![image](https://user-images.githubusercontent.com/1212885/178211753-961d6cd7-f63f-4ad9-af78-57557137c6f2.png)

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Microsoft Edge

### What version of Bootstrap are you using?

v5.2.0-beta1

**Comment by coliff** [comment]
thanks Patrick. I'll make a PR to add the aria-label. I don't think we need to add an additional note about it because it already has this near the top which covers it:

> - The `.progress-bar` also requires some `role` and `aria` attributes to make it accessible.

**Comment by patrickhlauke** [comment]
@coliff already started making a PR, so no worries :)

**Comment by coliff** [comment]
ah ok - great thanks. I got stuck at naming the aria-labels anyway :-D

**Comment by patrickhlauke** [comment]
ditto...i think i'm just going to go with something generic like "example" or similar. and something a bit more creative for the stacked bars example
